components/gc-features/gc-features-en.html

```diff
@@ -1,705 +1,206 @@
-					&lt;p&gt;[[comp:FeatureItem dct:description]]&lt;/p&gt;
-				&lt;/div&gt;
-			&lt;/div&gt;
-
-			&lt;!-- Column two --&gt;
-			&lt;div data-for="comp:FeatureItem in (comp:Feature comp:features )" class="col-lg-4 col-sm-6"&gt;
-				&lt;div class="well well-sm eqht-trgt"&gt;
-					&lt;img src="[[comp:FeatureItem html:img-src]]" alt="[[comp:FeatureItem html:img-alt]]"/&gt;
-					&lt;h3&gt;&lt;a class="stretched-link" data-bind="(href = comp:FeatureItem foaf:page)"&gt;[[comp:FeatureItem dct:title]]&lt;/a&gt;&lt;/h3&gt;
-					&lt;p&gt;[[comp:FeatureItem dct:description]]&lt;/p&gt;
-				&lt;/div&gt;
-			&lt;/div&gt;
-
-			&lt;!-- Column three [default] --&gt;
-			&lt;div data-for="comp:FeatureItem in (comp:Feature comp:features )" class="col-lg-4 col-sm-6"&gt;
-				&lt;div class="well well-sm eqht-trgt"&gt;
-					&lt;img src="[[comp:FeatureItem html:img-src]]" alt="[[comp:FeatureItem html:img-alt]]"/&gt;
-					&lt;h3&gt;&lt;a class="stretched-link" data-bind="(href = comp:FeatureItem foaf:page)"&gt;[[comp:FeatureItem dct:title]]&lt;/a&gt;&lt;/h3&gt;
-					&lt;p&gt;[[comp:FeatureItem dct:description]]&lt;/p&gt;
-				&lt;/div&gt;
-			&lt;/div&gt;
-
-		&lt;/div&gt;
-	&lt;/section&gt;</code></pre>
-</details>
-
-<h5>Cohabit design pattern</h5>
-
-<p>The GC Features component can be located on the right or on the left of the cohabit component by interchanging <code>col-md-8</code>  and <code>col-md-4</code>.</p>
-
-<p>To be used in this context:</p>
-<pre><code>&lt;div class="row mrgn-tp-xl"&gt;
-	&lt;div class="col-md-8"&gt;
-		&lt;section class="gc-features"&gt;
-			&lt;!-- gc-feature here --&gt;
-		&lt;/section&gt;
-	&lt;/div&gt;
-	&lt;div class="col-md-4"&gt;
-		&lt;!-- Cohabit component here. For example the Follow us component --&gt;
-	&lt;/div&gt;
-&lt;/div&gt;</code></pre>
-
-<h6>One feature tile - Cohabit design pattern</h6>
-<pre><code>&lt;div class="row mrgn-tp-xl"&gt;
-	&lt;div class="col-md-8"&gt;
-
-		&lt;section class="gc-features"&gt;
-			&lt;h2 class="wb-inv"&gt;[[comp:Feature dct:title]]&lt;/h2&gt;
-			&lt;div class="row" data-for="comp:FeatureItem in ( comp:Feature comp:features )"&gt;
-
-				&lt;!-- One column --&gt;
-				&lt;div class="col-md-6"&gt;
-					&lt;img class="thumbnail" src="[[comp:FeatureItem html:img-src]]" alt="[[comp:FeatureItem html:img-alt]]"/&gt;
-				&lt;/div&gt;
-				&lt;div class="col-md-6"&gt;
-					&lt;h3&gt;&lt;a class="stretched-link" data-bind="(href = comp:FeatureItem foaf:page)"&gt;[[comp:FeatureItem dct:title]]&lt;/a&gt;&lt;/h3&gt;
-					&lt;p&gt;[[comp:FeatureItem dct:description]]&lt;/p&gt;
-				&lt;/div&gt;
-			&lt;/div&gt;
-		&lt;/section&gt;
+</section>
 
-	&lt;/div&gt;
-	&lt;div class="col-md-4"&gt;
-		&lt;!-- Cohabit component here. For example the Follow us component --&gt;
-	&lt;/div&gt;
-&lt;/div&gt;</code></pre>
+<h2>Cohabit patterns (variant)</h2>
+<section>
```

Comment:
A section must contains an heading. Here (at this specific line) an implicit section is sufficient.